tools/topology/topology2/platform/intel/dmic-default.conf

```diff
@@ -19,4 +19,17 @@ Define {
 	DMIC1_NAME	dmic16k
 	DMIC0_PCM_CAPS	"Passthrough Capture 11"
 	DMIC0_PIPELINE_STREAM_NAME "copier.DMIC.12.1"
+	DMIC1_HOST_PIPELINE_ID		18
+	DMIC1_DAI_PIPELINE_ID		19
+	WOV_PIPELINE_ID			20
+	DMIC1_HOST_PIPELINE_SINK	'copier.host.18.1'
+	DMIC_WOV_DAI_PIPELINE_SRC	'copier.DMIC.19.1'
+	DMIC_WOV_DAI_PIPELINE_KPB	'kpb.19.1'
+	WOV_PIPELINE_SINK		'micsel.20.1'
+	WOV_PIPELINE_VIRTUAL		'virtual.detect_sink'
+	INCLUDE_WOV			'false'
+	WOV_UUID			'1f:d5:a8:eb:27:78:b5:47:82:ee:de:6e:77:43:af:67'
```

Comment:
add a comment here stating that this UUID corresponds to the test detect component?